_config.yml

```diff
@@ -109,7 +109,7 @@ defaults:
       path: "plugins/katello"
     values:
       layout: "plugins/katello/documentation"
-      latest: "3.15"
+      latest: "3.16"
```

Comment:
This (latest) shouldn't be changed until 3.16 goes GA. It actually means 'latest stable release'